fs/ftpfs.py

```diff
@@ -667,6 +668,18 @@ def getinfo(self, path, namespaces=None):
                 }
             )
 
+        if "modified" in namespaces:
+            if "basic" in namespaces or "details" in namespaces:
+                raise ValueError(
+                    'Cannot use the "modified" namespace in combination with others.'
+                )
```

Comment:
Why would you block `modified` from being used with the other namespaces? In particular, `basic` is always implied, so you can't really return an `Info` without the basic namespace.

In case of both `modified` and `details` being present, we could at most raise a warning, but the code should still work.

Reply:
I added that because I feel like requesting both "details" and "modified" at the same time is never a good idea to do intentionally. But I see your point; I'll change it to a warning.